kaspad/src/main.rs

```diff
@@ -17,7 +17,7 @@ pub fn main() {
     let _profiler = dhat::Profiler::builder().file_name("kaspad-heap.json").build();
 
     let args = parse_args();
-    let fd_total_budget = get_limit() - 64;
+    let fd_total_budget = limit() - args.rpc_max_clients as i32 - args.inbound_limit as i32 - args.outbound_target as i32;
```

Comment:
I think it's more readable to write `fd_budget::limit()` (here and in other places)